### PR TITLE
Ps6815

### PR DESCRIPTION
**Pull request by edwardstamps**
Hide "Name" Field and version updates

### CODE REVIEW

File: web/admin/src/App.css [hunk 1]
```diff
@@ -102,7 +102,7 @@ button.displayButton:focus, button.displayButton:active {
   background-color: white !important;
   line-height: 1;
   color: white;
-  text-shadow: 0 1px 0 #fff;
+  /* text-shadow: 0 1px 0 #fff; */
```
Comment (AdamLiechty) [inline web/admin/src/App.css]: remove?

File: web/admin/src/SurveyEditor.js [hunk 2]
```diff
@@ -107,7 +108,12 @@ class SurveyEditor extends Component {
           'multipletext',
         ],
       }
-      this.editor = new SurveyJSEditor.SurveyEditor('editorElement', editorOptions)
+      this.editor = new SurveyJSCreator.SurveyCreator('surveyCreatorContainer', editorOptions)
+      this.editor.onShowingProperty.add(function(sender, options) {
```
Comment (AdamLiechty) [inline web/admin/src/SurveyEditor.js]: this function looks like a duplicate of the one above.  extract.

File: web/admin/src/SurveyResults.js [hunk 3]
```diff
@@ -190,12 +190,15 @@ class SurveyResults extends Component {
...
-        .catch(err => result),
-    )
+    const attendeeClickPromises = results.map(result => {
+      if (result.creator.id) {
+        return this.props.client
+          .getAttendee(result.creator.id)
+          .then(attendee => ({ ...result, ...attendee }))
+          .catch(err => result)
+      }
+      return result
```
Comment (AdamLiechty) [inline web/admin/src/SurveyResults.js]: is `result` a Promise?

Comment (edwardstamps) [reply web/admin/src/SurveyResults.js]: It is not but this object needs to wait for the promises to finish. Is there a better way to structure this or would this cause issues?

Comment (AdamLiechty) [reply web/admin/src/SurveyResults.js]: 👍  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#Fulfillment

**Pull request merged**